pkg/volume/csi/csi_plugin.go

```diff
@@ -293,8 +294,14 @@ func initializeCSINode(host volume.VolumeHost) error {
 			Jitter:   0.1,
 		}
 		err = wait.ExponentialBackoff(initBackoff, func() (bool, error) {
+			err := kvh.WaitForNodeRegistrationCompleted(context.TODO())
```

Comment:
Not sure if it's useful to put this under exponential backoff since within it is waiting forever.